src/components/Filters/CompanyReceivedFilter.js

```diff
@@ -184,12 +184,14 @@ export const CompanyReceivedFilter = () => {
         {errors.length ? (
           <div className="a-form-alert a-form-alert--error" role="alert">
             {errors.map((message, key) => (
-              <div key={key}>
-                {message}
+              <>
```

Comment:
I think key should be moved to the wrapper container for this repeater so react doesn't throw a warning.